.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    es6: true,
+  },
+  extends: ['plugin:react/recommended'],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+  plugins: ['prettier', 'react'],
+  rules: {
+    'import/no-extraneous-dependencies': 'off',
+    'import/prefer-default-export': 'off',
+    'lines-between-class-members': ['error', 'always'],
+    'no-comma-dangle': 'off',
+    'no-multiple-empty-lines': ['error', { max: 1 }],
+    'no-unneeded-ternary': 'error',
+    'react/jsx-sort-props': 'error',
+    'react/jsx-uses-react': 'error',
+    'react/jsx-uses-vars': 'error',
+    'react/function-component-definition': 'off',
+    'react/jsx-one-expression-per-line': 'off',
+    'react/jsx-filename-extension': [
+      1,
+      {
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+      },
+    ],
+    'react/no-unknown-property': 'off',
```

Comment:
this can catch things like using class instead of className. Is there a reason why we want to turn it off?
`https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md`

Reply:
I'll remove the rule. It was just copied over from apollo.